edx_proctoring/views.py

```diff
@@ -975,8 +984,11 @@ def get(self, request, course_id, exam_id=None):
         Redirect to dashboard for a given course and optional exam_id
         """
         exam = None
+        attempt_id = None
         if exam_id:
             exam = get_exam_by_id(exam_id)
+            exam_id = exam['external_id']
```

Comment:
this may be slightly more clear without reusing a variable name (and, further, by renaming the parameter we pass this in as downstream)